bundles/XliffBundle/public/js/startup.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+pimcore.registerNS("pimcore.bundle.xliff.startup");
+
+
+pimcore.bundle.xliff.startup = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.preMenuBuild, this.preMenuBuild.bind(this));
+    },
+
+    preMenuBuild: function (e) {
+
+        let menu = e.detail.menu;
+        let that = this;
+        const user = pimcore.globalmanager.get('user');
+        const perspectiveCfg = pimcore.globalmanager.get("perspective");
+
+        if (user.isAllowed("translations") && perspectiveCfg.inToolbar("extras.translations")) {
```

Comment:
not sure if it should be independent from the `translations` permissions

Reply:
Yeah, I'd separate this from the translations. 👍

Reply:
we need to have a migration for enabling the new permission based on previous `translations` permission on users, roles & perspectives.